numbergame.py

```diff
@@ -23,12 +24,26 @@ def clearScreen(): #clears screen
     print("The rules of this game are this. \n\t Pick the lower and upper values of the range that the computer will pick the value in \n\t Guess the number, and it will tell you whether you arr too high, or too low. \n\t You want to guess the number in as little tries as possible. \n")
     input("Press ENTER to continue -->")
 
+LOLZ = None
+#LOLZ = True # Uncomment this line to add impossible mode...
+
 #finds out the range and then the number
-lowerNumber = int(input("\n\nWhat is the lower value for the range --> "))
-upperNumber = int(input("What is the upper value for the range --> "))
-number = random.randint(lowerNumber, upperNumber) #picks the number
+lowerNumber = input("\n\nWhat is the lower value for the range --> ")
+upperNumber = input("What is the upper value for the range --> ")
+if LOLZ:

```

Comment:
I could be wrong, but don't you need to do `if LOLZ == True`

Reply:
Nope. It's exactly why while True works:

`if` looks for a True. You can see this in action by running `5 == 5` - it will say True. If something returns True or is True then if will run the code. It's also why while True does an infinite loop. 

Additionally, it would be more accurate to do `if LOLZ is True`, if not `if LOLZ`.